alarm/nialarm.yml

```diff
@@ -307,6 +307,11 @@ definitions:
         description: AlarmId query. The service will return alarm instances whose *alarmId* fields are equal
           to the query, taking case into account.
         example: System.Health.DiskSpaceLow.Bdd9u!4aMd!$pYrf*CnaIZ2tbu$-Ct%?
+      description:
+        type: string
+        description: Description query. The service will return alarm instances whose *description* fields
+          contain the description query string, without taking case into account.
```

Comment:
Could we split the second sentence into two? I'd like the third sentence say something like, "This query does not take casing into account." This goes for all applicable instances.